lib/index.js

```diff
@@ -83,27 +83,34 @@ const getFilterExpression = (operator, property, value) => {
 		}
 
 		return row.typeOf().eq(getRethinkType(value)).default(false)
-	case 'const':
+	}
```

Comment:
What is all this?

Reply:
@LucianBuzzo https://eslint.org/docs/rules/no-case-declarations

Reply:
was failing on npm test because of eslint